installation_wizard_datachoice

```diff
@@ -0,0 +1,87 @@
+When installing ownCloud Server & ownCloud Enterprise editions the administrator
+may choose one of 3 supported database products.
+
+SQLite
+^^^^^^
```

Comment:
Thank you for your review. 
I used the ^^^^^^ per https://github.com/owncloud/documentation/blob/master/style_guide.rst
because I figured that "SQLite" should be a "Subhead, h3".

I notice that "Subhead, h2" and "Subhead, h3" seems to render the same.  However, if there is a future change to what ^^^^^^ means, "SQLite" is the third heading down from "Installation Wizard" being "Page Title, h1" and "Database Choice" being a "Subhead, h2".

As I said, I ran this through Sphinx and the output was as expected.  Before I change it to "=" or "-" as directed, I'd like a second opinion.